core/src/main/java/org/elasticsearch/index/IndexService.java

```diff
@@ -538,8 +537,9 @@ public void handle(ShardLock lock) {
         }
     }
 
-    public Settings getIndexSettings() {
-        return indexSettings;
+    @Override
+    public Settings indexSettings() {
```

Comment:
@s1monw: `#indexSettings()` is the overridden method from the base class and I wanted to reduce API surface area because both methods did the same. If I should reintroduce the getter, I would implement it identically to `#indexSettings()`, i.e.:

```
public Settings getIndexSettings() {
    return settingsService.getSettings();
}
```